deployments/container/Dockerfile

```diff
@@ -23,10 +23,23 @@ RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o ma
 
 # Use distroless as minimal base image to package the manager binary
 FROM nvcr.io/nvidia/distroless/go:v3.1.2
+
+ARG VERSION="unknown"
```

Comment:
Should we be passing the release version?

Reply:
yes, it is passed from the Makefile